### ISSUE TITLE
New update breaks the save features. 

### ISSUE THREAD

**Issue by vjdhama** [careteditor/caret#18]
The new caret isn't able to save.

The following error has occurred:
```
{
    "message": "Uncaught Error: ENOENT: no such file or directory, stat '/Users/vjdhama/Work/rails/crystal_lang.md'",
    "line": 854,
    "column": 18,
    "version": "0.4.3",
    "platform": "darwin",
    "os": "14.5.0",
    "path": "fs.js",
    "stack": [
        "at Error (native)",
        "at fs.statSync (fs.js:854:18)",
        "at Object.fs.statSync (ATOM_SHELL_ASAR.js:225:16)",
        "at FileManager.caret.inherit.saveFileContents (/Applications/Caret.app/Contents/Resources/app.asar/packages/FileManager/FileManagerDesktop.js:5:803)",
        "at FileManager.save (/Applications/Caret.app/Contents/Resources/app.asar/packages/FileManager/FileManager.js:5:2146)",
        "at loopEvents (/Applications/Caret.app/Contents/Resources/app.asar/web/Events.js:5:229)",
        "at Object.Events.trigger (/Applications/Caret.app/Contents/Resources/app.asar/web/Events.js:5:527)",
        "at emitTwo (events.js:87:13)",
        "at EventEmitter.emit (events.js:172:7)"
    ]
}
```

**Comment by erusev** [comment]
Thanks, we're working on it, we'll release a patch later today or sometime tomorrow.

**Comment by erusev** [comment]
This is fixed in the latest version (0.4.4).

You should get an automatic update.
